pyexebuilder/ExeBuilder.py

```diff
@@ -1,13 +1,16 @@
 __author__ = 'matth'
-import sys
-import os
-from py2exe.build_exe import py2exe as build_exe
-from py2exe.build_exe import Target
-import tempfile
-import subprocess
-from distutils.dist import Distribution
 import ctypes.util
+import os
 import shutil
+import subprocess
+import sys
+import tempfile
+
+if sys.version_info[0] == 2:
```

Comment:
So the differences are mostly due to changes in py2exe-0.6 vs py2exe-0.10 rather than Python 2 vs Python 3. So it may be better to branch on the py2exe version rather than the Python version. But it feels simpler to branch on the Python version.

What do you think?

Reply:
We are updating to py2exe-0.10 to add support for Python 3, so I am happy to have this as past of the Python 3 changes.